src/cljs/zetawar/data.cljs

```diff
@@ -18,7 +18,6 @@
 
     ;; Faction configuration
     :configure-faction-tip"Configure faction"
-    :configure-faction-title "Configure faction"
```

Comment:
I'd keep this and rename it to :configuration-faction-title-prefix. Also, I prefer "Configure faction" to "Configure Faction". I'm not a big fan of title case.